packages/nimble-components/src/wafer-map/styles.ts

```diff
@@ -2,18 +2,25 @@ import { css } from '@microsoft/fast-element';
 import { borderColor, borderWidth } from '../theme-provider/design-tokens';
 
 export const styles = css`
+    :host {
+        display: inline-block;
+        width: 100%;
+        height: 100%;
```

Comment:
This host styling is the default size of a wafer-map if placed on a page with just:

```
<nimble-wafer-map></nimble-wafer-map>
```

It should be some reasonable, arbitrary size. Keeping width: 500px; and width: 500px; would be fine if not a tad big. I might suggest 300x300 instead.

Then users can update the size using css as expected:

```css
nimble-wafer-map {
    width: 100%;
    height: 100%;
}
```

Reply:
applied